server/src/interfaces/notification.interface.ts

```diff
@@ -70,18 +71,22 @@ export type EmailRenderRequest =
   | {
       template: EmailTemplate.TEST_EMAIL;
       data: TestEmailProps;
+      customTemplate?: string;
```

Comment:
Should this be optional for all template?

Reply:
I think it would be good to provide this for all templates already, even if they don't have an ui to override it. That way all templates behave the same.

Reply:
Do you mean that this property will still be used for other email types, even without a template?

Reply:
Yes indeed

Reply:
Then, should the property here be specified as required instead of optional?